docs/how-to-create-a-new-release.rst

```diff
@@ -103,27 +103,25 @@ still work on a new installation. This is important because it can find missing
 Building the documentation
 --------------------------
 
-An important part of the release process is ensuring that the documentation for readthedocs_ site can be built. This can
-be tested locally by executing the following commands in the repository folder::
+Well documented code is an essential part of the release, allowing your code's legacy to only grow in glory and admiration.
+The documentation will be rendered via the readthedocs_ site, allowing future developers, practitioners, researchers and students
+to understand and build upon your work. Note: entry points created by the command ``cea install-toolbox`` should be run at least once
```

Comment:
I don't understand "entry points created by the command ``cea install-toolbox`` should be run at least once". What entry points are you referring to? Also, note that we have moved the ArcGIS interface to legacy and `install-toolbox` is no longer supported.

Reply:
yeah this was in the old documentation and I wasn't sure if we still needed it.